docs/content/intro/install.md

```diff
@@ -79,9 +79,5 @@ Brigade is well-tested on [AKS Kubernetes](https://docs.microsoft.com/en-us/azur
   DNS service or another DNS service) to your Load Balancer IP. GitHub and other
   webhook services seem to work better with DNS names than with IP addresses.
 
----
-
-Prev: [Overview][overview] `|` Next: [Writing your first CI pipeline, Part 1][part1]
-
-[overview]: overview.md
-[part1]: tutorial01.md
\ No newline at end of file
+[overview]: ../overview
```

Comment:
Was the `../` added to work with the presented website?  The relative linking no longer works locally (code editor) with this path change, but definitely defer to it working in live form (brigade.sh).

Reply:
Exactly, the `../` was added (educated guess :D) since I found out that this is the way `docs.brigade.sh` mechanism works. Not sure how to test it, so I'll ping @flynnduism for another review of this!

Reply:
Because those custom navs links were removed from the end of the page, these 'footnote' type links can also be removed as they are not included anywhere else on this page.

In some cases they are still used for links in the main article, but not here.